src/classes/VectorLayer.js

```diff
@@ -156,7 +156,7 @@ class VectorLayer extends olLayerVector {
     if (geomType instanceof olGeomPoint || geomType instanceof olGeomMultiPoint) {
       style = [new olStyleStyle({
         image: new olStyleCircle({
-          fill: new olStyleFill({ color: 'rgba(255,255,255,1)' }),
+          fill: new olStyleFill({ color: 'rgba(255,255,255,0.4)' }),
```

Comment:
Is the change to 0.4 opacity an expected change here?

Reply:
Yeah, it's to better match the default openlayers style and as a conveniance so that drawn shapes don't occlude useful data by default.

Reply:
OpenLayers default styles: https://openlayers.org/en/latest/apidoc/module-ol_style_Style-Style.html